templates/hpa.yaml

```
@@ -4,14 +4,8 @@
 apiVersion: autoscaling/v2
 kind: HorizontalPodAutoscaler
 metadata:
-  {{- if hasKey $serviceDefinition.hpa "name" }}
-  name: {{ $serviceDefinition.hpa.name }}
-  {{- else }}
   name: {{ include "common.fullname" (dict "root" $ "service" $serviceDefinition "serviceName" $serviceName ) }}-hpa
```

No need to postfix :-)
```suggestion
  name: {{ include "common.fullname" (dict "root" $ "service" $serviceDefinition "serviceName" $serviceName ) }}
```